svelte/components/Nft/Nft.svelte

```diff
@@ -3,248 +3,229 @@
 
   import type { NftType } from "lib/ktypes";
   import {
-    getShortAddress,
     nftUrl,
     explorerCollectionUrl,
-    nftDescription,
-    nftDescriptionShort,
-    nftName,
-    nftOpenSeaUrl,
-    addressSame,
-    textShort,
     explorerAddressLink,
     kredeumNftUrl,
-    getNetwork
+    getNetwork,
+    nftOpenSeaUrl,
+    addressSame
   } from "lib/kconfig";
-  // import { nftKey } from "lib/kconfig";
-  import { nftGetImageLink } from "lib/knft-get-metadata";
-  import { onMount } from "svelte";
+
+  import { shortcode } from "helpers/shortcodes";
 
   import { nftStore } from "stores/nft/nft";
+
   import NftTransfer from "./NftTransfer.svelte";
 
-  /////////////////////////////////////////////////
-  //  <Nft {chainId} {address} {tokenID} {account}? {index}? {more}? {platform}? />
-  // Display NFT
-  /////////////////////////////////////////////////
   export let chainId: number;
   export let address: string;
   export let tokenID: string;
   export let account: string = undefined;
-  export let index = 0;
-  export let more = 0;
   export let platform: string = "dapp";
-
   let nft: Readable<NftType>;
 
   // let i = 1;
   // HANDLE CHANGE : on truthy chainId and address, and whatever account
   $: account, chainId && address && tokenID && handleChange();
-  const handleChange = (): void => {
-    // console.log(`NFT CHANGE #${i++} ${nftKey(chainId, address, tokenID)}`);
+  const handleChange = async (): Promise<void> => {
```

Comment:
async unnecessary, shortcode missing : re-use previous code

Reply:
The async is for the line 38 : await nftStore.refreshOne(chainId, address, tokenID);
You think we should not refresh because at this moment the nft is necessarily loaded?

Reply:
I've put the shortcode in a shortcodes.ts file in the helpers folder, because the function is called from Nft.svelte and NftLine.svelte
i wrote shortcodes as plural because in Mr GD's nftDetail model there was a modal which propose different shortcodes